utam-core/src/main/java/utam/core/selenium/element/ElementAdapter.java

```diff
@@ -183,6 +184,11 @@ public void deprecatedClick() {
     driverAdapter.executeScript(CLICK_VIA_JAVASCRIPT, getWebElement());
   }
 
+  @Override
+  public byte[] getScreenshot() {
+    return webElement.getScreenshotAs(OutputType.BYTES);
```

Comment:
nit: pls use getWebElement() for consistency with other methods

Reply:
Good catch. Fixed in subsequent commit.